### ISSUE TITLE
buy_pct on original value?

### ISSUE THREAD

**Issue by ghost** [DeviaVir/zenbot#525]
buy_pct  keeps taking parts of remaining balance rather than the percentage of the original balance. EG if start balance is $50 and buy_pct is set to 10%. It will take 5$ then, 4.5$ then 4.05$ etc. It should take 5$ from 50 each time, and then stop once there is no balance remaining.

**Comment by DeviaVir** [comment]
duplicate https://github.com/carlos8f/zenbot/issues/174

**Comment by DeviaVir** [comment]
no, I should correct myself, the question here is to keep the original value in mind. Sounds like a potentially unwanted enhancement.

**Comment by crack00r** [comment]
any news on this?

**Comment by DeviaVir** [comment]
I think some functionality like this landed recently, please take another look and let me know if this issue is still accurate.
